src/odemis/driver/piezomotor.py

```diff
@@ -787,6 +856,79 @@ def _openSerialPort(port):
 
         return ser
 
+    @classmethod
+    def scan(cls):
+        """
+        returns (list of 2-tuple): name, kwargs
+        Note: it's obviously not advised to call this function if a device is already under use
+        """
+        if os.name == "nt":
+            raise NotImplementedError("OS not yet supported")
+        else:
+            ports = list(serial.tools.list_ports.grep('/dev/ttyUSB*'))
```

Comment:
What I have in mind is that you can replace all this by:
```python
ports = [p.device for p in serial.tools.list_ports.comports()]
```
Is there a problem with using `comports()`?